network/web3bridge.js

```diff
@@ -179,55 +207,63 @@ class Web3Bridge {
         }
     }
 
-    async getPastEvents(target, contractName, event, options={fromBlock: 0, toBlock: 'latest'}) {
+    async getPastEvents(target, contractName, event, options = {fromBlock: 0, toBlock: 'latest'}) {
         const contract = await this.loadWebsiteContract(target, contractName);
         return await contract.getPastEvents(event, options);
     }
 
-    async getBlockTimestamp(blockNumber){
+    async getBlockTimestamp(blockNumber) {
         return (await this.web3.eth.getBlock(blockNumber)).timestamp;
     }
 
-    async subscribeContractEvent(target, contractName, event, onEvent, onStart, options={}) {
+    async subscribeContractEvent(target, contractName, event, onEvent, onStart, options = {}) {
         const contract = await this.loadWebsiteContract(target, contractName);
 
         let subscriptionId;
         const subscription = await contract.events[event](options)
-            .on('data', data => onEvent({ subscriptionId, data }))
-            .on('connected', id => onStart({
-                subscriptionId: subscriptionId = id,
-                data: {
-                    message: `Subscribed to "${contractName}" contract "${event}" events with subscription id: ${id}`
-                }
-            }));
+            .on('data', data => onEvent({subscriptionId, data}))
+            .on('connected', id =>
+                onStart({
+                    subscriptionId: (subscriptionId = id),
+                    data: {message: `Subscribed to "${contractName}" contract "${event}" events with subscription id: ${id}`}
```

Comment:
Is this a `prettier` bug? The line length here is 125. So it didn't fix anything, just made the code less readable.